### DISCUSSION TITLE
New accessory type "Solar"

### DISCUSSION THREAD

**Discussion by vgr28** [arachnetech/homebridge-mqttthing#599] [General]
Hi, is it possible to add new accessory type "Solar" with the following topics?

 topics = 
    'current',
    'voltage',
    'power',
    'powerdc',
    'yieldtotal',
    'powerfactor',
    'frequency',
    'temperature',
    'settings/limit',

**Comment by mastertape** [comment]
would be great also , may be a solution to ad custum fields
